feat: design updates [RWDQA-62]

tomzemp opened the pull request:
See ticket: https://dhis2.atlassian.net/browse/RWDQA-62

This PR updates design 
- replaces Menu Bar with navigation buttons
- uses splash screens (logo for report page, instructions for configurations)
- adds tooltip to generate report button when disabled

Screenshots:
<img width="1420" alt="image" src="https://github.com/hisprwanda/who-data-quality-annual-report/assets/18490902/fbd68541-9572-43b2-b205-500773b446bf">
<img width="1416" alt="image" src="https://github.com/hisprwanda/who-data-quality-annual-report/assets/18490902/51d595f8-7476-4814-9d5a-22f76254f134">


**Note!** 

Here I have not totally followed Joe's design and kept the context select bar on the configurations page. I prepared a branch where I did that, but in order to move <ReportParameterSelector> such that it is constant over the pages, I had to move it outside of the router, which leads to [some difficulties in getting links to pages](https://github.com/hisprwanda/who-data-quality-annual-report/blob/move-context-select-bar/src/components/report-parameter-selector/ReportParameterSelector.js#L32-L55) to work (among other things). The<ReportParameterSelector> also does not show selections when the <SelectBarItem>s are disabled, so the UX benefit is perhaps only limited in any case?

![with_constant_select_bar](https://github.com/hisprwanda/who-data-quality-annual-report/assets/18490902/8b3d00f9-2b2d-49b5-a6b4-32284c93d085)

Anyway, the difference between this PR and the branch with the fixed <ReportParameterSelector> is in this [commit](https://github.com/hisprwanda/who-data-quality-annual-report/commit/df43426e444b47f006f8209039a95b2cf81497eb). I'm voting to not implement that, but would happily take feedback here.

tomzemp commented:
Following up on suggestions from meeting today, I've moved the "Exit Configurations" button:
<img width="1192" alt="image" src="https://github.com/hisprwanda/who-data-quality-annual-report/assets/18490902/800aa6cd-3c0a-4f34-b505-d92e4c46d7c0">

tomzemp commented on src/components/config-tabs/ConfigTabs.module.css:
```diff
@@ -2,74 +2,17 @@
-}
-.content p {
+.exitConfigurationsContainer {
+    height: 48px;
```
I had to set the height here directly because `height: 100%` was not working. I guess because the parent TabBar div does not have a known height?

tomzemp commented on src/components/config-tabs/ConfigTabs.js:
```diff
@@ -10,176 +14,95 @@ import { ExternalDataComparison } from './tab-contents/ExternalDataComparison.js
-                        mappedNumerators={mappedNumerators}
-                    />
+        <>
+            <TabBar>
```
ideally this would be `scrollable`, but when I add that property, I get errors (`ResizeObserver loop completed with undelivered notifications.`) These probably don't occur in production, but I'm not sure it makes sense to add for now 🤔?

KaiVandivier reviewed (approved):
Looks great! Thanks for doing this! 🎉 🚀 

Just wrote a few optional suggestions 🙂

tomzemp commented:
Thanks @KaiVandivier! I've cleaned up the css per your suggestions and have also added the logo on the configurations page per @mauricejulesm's suggestion from the meeting today
